examples/keep-alive-framework/build.gradle

```diff
@@ -14,3 +16,14 @@ dependencies {
     implementation project(':test-utils')
     implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.1.7'
 }
+
+task uploadTarball(type: S3Upload) {
+    description = "Upload the framework tarball to S3."
+    group = "CI"
+    dependsOn distTar
+
+    region = "us-west-2"
+    bucket = "usi-builds"
+    prefix = "mesosphere/usi/artifacts"
+    source = files(distTar.archiveFile).asFileTree
+}
```

Comment:
We might want to investigate [Gradle Publishing](https://docs.gradle.org/current/userguide/distribution_plugin.html#sec:publishing_distributions) but this should suffice to get the latest master build on Soak.

Reply:
I'd love to trigger a restart on soak but it does not seem trivial to authenticate. Checkout https://github.com/mesosphere/soak-cluster-configurations/pull/506 for the app.